.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,71 @@
+---
+name: Enhancement
+description: Suggest an enhancement to existing functionality
+title: "[Enhancement]: <Title>"
+labels: [ "enhancement" ]
+body:
+  - type: dropdown
+    id: type
+    attributes:
+      label: What type of enhancement is this?
+      multiple: true
+      options:
+        - Performance
+        - User experience
+        - API improvement
+        - Configuration
+        - Refactor
+        - Other
+    validations:
+      required: true
+
+  - type: dropdown
+    id: subsystem
+    attributes:
+      label: What subsystems and features will be improved?
+      description: You can pick multiple subsystems and features.
+      multiple: true
+      options:
+        - Query planner
+        - Query executor
+        - Continuous aggregate
+        - Background worker
+        - Policy
+        - User-Defined Action (UDA)
```

Comment:
This label is currently `custom-action` so we should probably change the name of that one if we use this label.

Reply:
Note that these aren't translated into labels so there is no 1-1 match right now. Also, if we add a job to turn these into labels, we need not have exactly matching names anyway.

I think priority here is to be user descriptive, rather than matching labels since users will add these.

Reply:
Yes, noted that, but it is good for our own usage that the match between the options and the labels are similar. For example, "User Defined Actions" and "Custom Actions" are quite different and it will be confusing if we do not try to keep them similar.

Reply:
We can change labels at any time and it does not affect this part of the PR. Again, the thing in this list is _not_ a label and User-defined Action is the official term that we also use in docs.

Reply:
So change the name of the label?

Reply:
Sure, but what does it have to do with this PR? Especially since this is _not_ a list of labels? We don't have labels for many things on this list.

Reply:
It has nothing to do with this PR? I already approved it and you can push it, but I just added that you might want to change the labels to match the options to avoid confusion when triaging?